jenkins-scripts/dsl/ignition.dsl

```diff
@@ -201,14 +230,22 @@ ArrayList all_debbuilders()
   List<String> branches = new ArrayList<String>();
   // add all supported branches
   ignition_software.each { ign_software ->
-    supported_branches("${ign_software}").each { major_version ->
+    supported_ign_branches("${ign_software}").each { major_version ->
       if (major_version) {
         // No 1-debbuild versions, they use the unversioned job
         if ("${major_version}" == "0"  || "${major_version}" == "1" )
           major_version = ""
           branches.add("ign-${ign_software}${major_version}")
       }
     }
+    supported_gz_branches("${gz_software}").each { major_version ->
+      if (major_version) {
+        // No 1-debbuild versions, they use the unversioned job
+        if ("${major_version}" == "0"  || "${major_version}" == "1" )
```

Comment:
We don't have any libraries in this situation, but I imagine it could happen in the future if we add a new library